bin/ingest.py

```diff
@@ -245,8 +245,8 @@ def main(args):
             os.makedirs(pdfs_dest_dir)
 
         # copy the book
-        book_src_filename = meta["abbrev"] + "-" + year
-        book_src_path = os.path.join(root_path, book_src_filename) + ".pdf"
+        book_src_filename = f'{year}.{meta["abbrev"]}-{volume_name}.pdf'
+        book_src_path = os.path.join(root_path, book_src_filename)
         book_dest_path = None
```

Comment:
We were using the wrong pattern to find the books, with the result that many recent books were not committed. This fixes that.